pwiz_tools/Skyline/CommandArgUsage.resx

```diff
@@ -928,4 +928,7 @@ greater peak area.</value>
   <data name="_tran_use_dia_window_exclusion" xml:space="preserve">
     <value>Take precursor exclusion window from the Full-Scan settings instead of being symmetric around the precursor m/z. This is recommended for DIA/SWATH data.</value>
   </data>
+  <data name="CommandArgs_ARG_FULL_SCAN_PRODUCT_ISOLATION_SCHEME_path_to_result_file_to_import_the_isolation_scheme" xml:space="preserve">
+      <value>path/to/result-file to import the isolation scheme</value>
```

Comment:
Does this read smoothly without the "from"?

Reply:
Personally, I preferred the version that included the word "from".

Without the "from", it certainly sounds like this has something to do with importing an isolation scheme into a result file (whatever that would mean).

If the description had been "name/of/nation to import the BMWs" I would think it was expecting "United States" instead of "Germany".

Reply:
Well, in this case, I was thinking the destination (the document settings) was obvious. But, go for it. You can put " from" back in.